_episodes/07-cond.md

```diff
@@ -98,6 +98,18 @@ else:
 Note that to test for equality we use a double equals sign `==`
 rather than a single equals sign `=` which is used to assign values.
 
+> ## Comparing in python
```

Comment:
For consistency, we should capitalize "Python".

Reply:
Should we say what we're comparing here (numbers)? Or should we leave it vague because later they might learn about `"a" < "b"` ?

Reply:
I like keeping it general, since the operators can be used with other types than numbers as you point out.